en/02_Developer_Guides/00_Model/11_Scaffolding.md

```diff
@@ -83,6 +83,91 @@ You can also alter the fields of built-in and module `DataObject` classes by imp
 > [!NOTE]
 > `FormField` scaffolding takes [`$field_labels` config](#field-labels) into account as well.
 
+## Scaffolding for relations
+
+Form fields are also automatically scaffolded for `has_one`, `has_many`, and `many_many` relations. These have sensible default implementations, and you can also customise what form field will be used for any given `DataObject` model.
```

Comment:
```suggestion
Form fields are also automatically scaffolded for `has_one`, `has_many`, and `many_many` relations on the form created when editing DataObjects, with the notable exception of the [SiteTree](api:SilverStripe\CMS\Model\SiteTree) edit form. These have sensible default implementations, and you can also customise what form field will be used for any given `DataObject` model.
```

Reply:
I've put it in the second above this instead, since that isn't something unique to fields scaffolded for relations.